.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+### Suggested package name(s)
+<!-- Name suggested package(s) -->
+
+### Alternative to
+<!-- Name popular package(s) the suggested one is an alternative to -->
+
+### Why is the suggested package better?
+<!-- Please explain here -->
+
+### Checklist
+- [ ] Package has sufficient overlap in functionality to act as a replacement.
+- [ ] Package is a better alternative to what is already suggested for this category, or the category is new.
+- [ ] Package is not deprecated and actively maintained (the last release was published less than 6 months ago).
+- [ ] Package has at least 1000 weekly downloads on NPM.
+- [ ] Package has 100+ stars on GitHub.
```

Comment:
There are packages that don't have a lot of github stars (because they might be nice / micro libs that don't have a community), but a large number of NPM downloads (because folks directly find these there). This was the original motivation for keeping an "OR" between these.